### PR TITLE
completed database

### PR DESCRIPTION
**Pull request by az9222**

### CODE REVIEW

File: client/public/bundle.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+/************************************************************************/
+/******/ ([
+/* 0 */
+/***/ (function(module, exports, __webpack_require__) {
+
+"use strict";
+
+
+/***/ })
+/******/ ]);
```
Comment (alexsheehan) [inline client/public/bundle.js]: Needs newline

File: client/public/index.html [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  <title>countdown</title>
+</head>
+<body>
+  <div id="countdown"></div>
+  <script crossorigin src="https://unpkg.com/react@16/umd/react.development.js"></script>
+  <script crossorigin src="https://unpkg.com/react-dom@16/umd/react-dom.development.js"></script>
+  <h1>Review</h1>
+</body>
+<script type="text/javascript" src="./client/public/bundle.js"></script>
+</html>
```
Comment (alexsheehan) [inline client/public/index.html]: Needs newline

File: database/index.js [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+  } else {
+    console.log("Connection made.");
+    //the moment the db connects, we start adding info into restaurants
+    for (let i = 0; i < 15; i++) {
+      let restaurantName = faker.company.companyName();
+      insertIntoRestaurant(restaurantName, (err, results) => {
+        if (err) {
+          console.log("Error:", err);
+        } else {
+          console.log(`${restaurantName} added successfully`);
```
Comment (alexsheehan) [inline database/index.js:22]: console.logs in this file could get annoying

File: schema.sql [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+CREATE TABLE restaurant (
+  restaurant_id INT NOT NULL AUTO_INCREMENT,
+  name VARCHAR(50) NOT NULL,
+  PRIMARY KEY (restaurant_id)
+);
+
+CREATE TABLE user (
+  user_id INT NOT NULL AUTO_INCREMENT,
+  username VARCHAR(50) NOT NULL,
```
Comment (alexsheehan) [inline schema.sql:13]: What is the username if they provide an anonymous review - if null, why is this 'NOT NULL'?

File: server/index.js [hunk 5]
```diff
@@ -0,0 +1,105 @@
+const express = require("express");
```
Comment (alexsheehan) [inline server/index.js:1]: If you're not ready to push this file, it shouldn't be in the commit/PR

File: server/index.js [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+//     const schema = {review: joi.string().min(20).required()};
+//     if (error) {
+//       res.status(404).send("review not found");
+//     } else {
+//       res.status(200).send(foundReview);
+//     }
+//   });
+// });
+
+app.listen(port, () => {console.log(`Listening to port ${port}...`)});
```
Comment (alexsheehan) [inline server/index.js:109]: Needs newline

File: webpack.config.js [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+          presets: ['react', 'es2015', 'env']
+        },
+      },
+    ],
+  },
+  output: {
+    path: __dirname + '/public',
+    filename: 'bundle.js',
+  }
+};
```
Comment (alexsheehan) [inline webpack.config.js]: Needs newline

**Review by alexsheehan — CHANGES_REQUESTED** [review]

File: client/public/index.html (hunk 2 above)
Comment (az9222) [reply client/public/index.html]: What do you mean by the newline comments? Thanks!

Comment (alexsheehan) [reply client/public/index.html]: Git gives an error message at the end of files if they don't end in a new line - https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

**Pull request merged**